cli-setup.sh

```diff
@@ -145,15 +145,32 @@ setup_venv() {
     fi
 }
 
+prompt_account_inputs() {
+    read -p "Platform9 account management URL [Example: https://example.platform9.io]: " MGMTURL
+    if [[ ${MGMTURL} != https://* ]]; then
```

Comment:
this will break if someone enters http://, it will become "https://http://hello.platform9.net"

Reply:
Am ok with it right now. Maybe the example will help. If I see people making that mistake, I can adjust the script to handle it. (I try to avoid writing too much stuff in bash if I can avoid it :) )